ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14__openshift-pipelines-ocp4.16-lp-p2p.yaml

```diff
@@ -0,0 +1,63 @@
+base_images:
+  cli:
+    name: "4.16"
```

Comment:
should be 4.15 released OCP version

Reply:
Thanks for catching that. Switching it to ROSA right now. Will make that chang